arbitrum-docs/how-arbitrum-works/timeboost/timeboost-faq.mdx

```diff
@@ -0,0 +1,183 @@
+
+### How does Timeboost affect Orbit chains that settle to Arbitrum One or Arbitrum Nova?
+
+will introduce a (proposed) 200 ms delay on non-express lane transactions and grant a time advantage/privilege of immediate sequencing of transactions sent by the auction winner. This delay is a new transaction ordering policy at the sequencer level, and, as with many other Nitro tech features, this will be available for Orbit chains to adopt, including those that settle on Arbitrum One or Arbitrum Nova.
+
+However, there's probably no meaningful time advantage for the L3's batched transactions data or assertion here. There could be a world where an Orbit chain is incredibly time-sensitive and participates in the Timeboost auctions because they want a faster confirmation that their data and/or assertion gets sequenced on the parent chain. Still, we're talking about a 200 ms difference between the non-express and express lanes.
+
+### Does this vote to pass automatically mean that all orbit chains will have Timeboost implemented?
+
+No, Timeboost implementations are at the sequencer level, so if the DAO passes this vote, Timeboost will roll out to the sequencer for Arbitrum One and Nova. This approach is the same strategy for ArbOS upgrades since Orbit chains can choose to adopt Timeboost if they want to - of the voting outcome for adoption on Arbitrum One and Nova.
+
+Technically, the vote passing will mean Orbit L3s settling to Arbitrum One, and Nova will have their batched transaction data and assertions in the parent chain's non-express lane or express lane. Still, Orbit teams can decide whether they want their Orbit chain's sequencer to implement Timeboost. Your chain, your rules.
+
+### Does it mean we expect searchers to bid continuously in advance, expecting opportunities to happen 1 minute later, rather than "in real time" opportunities (I see something → I submit an arbitrage tx with priority)?
+
+Before answering this question, it is worth clarifying that the participant will likely attempt to predict the amount of MEV generated between 15s and 1min 15s in the future - not 1 minute later. This prediction is because the auction is closed and resolved at a maximum of 15 seconds before the start of the next round (in the current proposal).
+
+Participants will bid continuously for the right to use the express lane in advance so that they (the participant) can profit from both (1) MEV opportunities they predict between 15s and 1min 15s in the future and (2) MEV opportunities in real-time during the period that the participant is in control of the express lane that they didn't otherwise predict in advance (proposed duration: 1 minute). If the participant does not win control of the express lane, opportunities that they see in real-time are still exploitable, but with a 200ms delay like all other txs (since only the express lane controller's transactions get sequenced with no delay).
```

Comment:
```suggestion
Participants will bid continuously for the right to use the express lane in advance so that they (the participants) can profit from both (1) MEV opportunities they predict between 15s and 1min 15s in the future and (2) MEV opportunities in real-time during the period that the participant is in control of the express lane that they didn't otherwise predict (proposed duration: 1 minute). Suppose the participant does not win control of the express lane. In that case, opportunities they see in real-time are still exploitable, but with a 200ms delay like all other transactions (since only the express lane controller's transactions get sequenced with no delay).
```